EventFilter/TotemRawToDigi/test/analyze_vfat_frames.py

```diff
@@ -0,0 +1,43 @@
+import FWCore.ParameterSet.Config as cms
+import FWCore.ParameterSet.VarParsing as VarParsing
+
+process = cms.Process("AnalyzeVFATFrames")
+
+# default options
+options = VarParsing.VarParsing ('analysis')
+options.inputFiles= 'file:/afs/cern.ch/user/j/jkaspar/public/run268608_ls0001_streamA_StorageManager.root',
```

Comment:
RAW files are in permanent storage on tapes.
Copies can be pulled to disk on request. 
The fact that I was able to read the file means that it's still on disk, but that could change.

Please send an email to David and Daniele to sort out if some group quota can be setup for you in the CMS storage phase space.